### DISCUSSION TITLE
How to prevent shifting the input value to left when user delete a part of input value. (Like date or IP address)

### DISCUSSION THREAD

**Discussion by nagisaando** [beholdr/maska#137] [Q&A] [answered]
Hello, I have a date mask input. As this; dd/mm/yyyy

For example, after I filled the input as 22/12/2002 and I deleted the part of day (22) and changed the date to 10, 

The current behavior:
It pushes the all rest of the input value to left
```
12/20/02__
```
I expect this is expected behavior, but I also found it a bit confusing to the user. Is there any way to keep it blank instead of pushing the all input value to left when the user deletes a part of the input?

The ideal behavior: 
```
__/12/2002
```

I could not find the native feature of Maska to support this behaviour, but if there is any workaround, it would be great. Thanks for your help!

**Comment by beholdr** [comment, marked as answer]
I'm afraid it’s not possible with Maska. For this to work you need mask library with placeholder support.

**Reply by nagisaando** [reply]
Thank you for the answer. Yes, I was thinking it would be difficult. Thank you!
